### PR TITLE
ME15: Added LP OVR Feature Support.

### PR DESCRIPTION
**Pull request by sihyung-maxim**
Added LP Operating Voltage Range Select Support.

### CODE REVIEW

File: Libraries/PeriphDrivers/Source/LP/lp_me15.c [hunk 1]
```diff
@@ -102,9 +104,94 @@ void MXC_LP_EnterShutDownMode(void)
...
+            MXC_GCR->memctrl = (MXC_GCR->memctrl & ~(MXC_F_GCR_MEMCTRL_FWS)) |
+                               (0x2UL << MXC_F_GCR_MEMCTRL_FWS_POS);
+
+        } else {
+            MXC_GCR->memctrl = (MXC_GCR->memctrl & ~(MXC_F_GCR_MEMCTRL_FWS)) |
+                               (0x1UL << MXC_F_GCR_MEMCTRL_FWS_POS);
+        }
+    }
+
+    // Caller must perform peripheral reset
```
Comment (Jake-Carter) [inline Libraries/PeriphDrivers/Source/LP/lp_me15.c:192]: Document this in the Doxygen header as well

**Review by Jake-Carter — APPROVED** [review]
Looks good, one minor note below

File: Libraries/PeriphDrivers/Source/LP/lp_me15.c (hunk 1 above)
Comment (sihyung-maxim) [reply Libraries/PeriphDrivers/Source/LP/lp_me15.c:192]: Done.

**Comment by sihyung-maxim** [comment]
Something is wrong with checking out the repo in the BLE_Examples_Test workflow (GitHub side issue). These changes aren't related to the BLE tests either and after talking with Eddie, it should be good to merge in even if the BLE test fails.

File: Libraries/PeriphDrivers/Source/LP/lp_me15.c [hunk 2]
```diff
@@ -102,9 +104,94 @@ void MXC_LP_EnterShutDownMode(void)
...
+        }
+    }
+
+    // Set flash wait state for any clock so its not to low after clock changes.
+    MXC_GCR->memctrl = (MXC_GCR->memctrl & ~(MXC_F_GCR_MEMCTRL_FWS)) |
+                       (0x5UL << MXC_F_GCR_MEMCTRL_FWS_POS);
+
+    // Set the OVR bits
+    MXC_PWRSEQ->lpcn &= ~(MXC_F_PWRSEQ_LPCN_OVR);
+    MXC_PWRSEQ->lpcn |= ovr;
```
Comment (Jacob-Scheiffler) [inline Libraries/PeriphDrivers/Source/LP/lp_me15.c:132]: I'd suggest using MXC_Setfield here to ensure ovr doesn't overwrite any other field in that register or check that it's a valid selection beforehand.

**Review by Jacob-Scheiffler — APPROVED** [review]

**Pull request merged**